components/automate-cli/pkg/verifyserver/utils/fiberutils/helper.go

```diff
@@ -0,0 +1,42 @@
+package fiberutils
+
+import (
+	"net/url"
+
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/models"
+	"github.com/sirupsen/logrus"
+)
+
+func GetHostFormEndPoint(endpoint string) (string, error) {
+	parsedURL, err := url.Parse(endpoint)
+	if err != nil {
+		logrus.Errorf("cannot parse the endpoint, error: %+v", err)
+		return "", err
+	}
+
+	return parsedURL.Hostname(), nil
+}
+
+// ConstructIpAndNodeTypeMap prepare the map with key as IP address and value as node type
+func ConstructIpAndNodeTypeMap(config models.Config) map[string]string {
+	ipNodeMap := make(map[string]string)
+	// For automate nodes
+	for _, ip := range config.Hardware.AutomateNodeIps {
```

Comment:
We need bastion also over here. 

<img width="579" alt="image" src="https://github.com/chef/automate/assets/91940132/8004542d-acd2-43ca-9019-c3c0280fac6e">

Reply:
handling as a separate commit